api/descriptions/tags/tag_cve_post.md

```diff
@@ -18,6 +18,18 @@ Prisma Cloud ingests all the various distro vulnerability feeds, and normalizes
 
 The **package info** tab shows both source and binary package fields in a vulnerability report.
 
+Refer to the following parameter descriptions:
+- **id**: `Required` Specifies the vulnerability ID.
```

Comment:
@ssugandh More specifically, this is a CVE ID. I think we should be clear about that here.

As a reference, see here: https://cve.mitre.org/cve/identifiers/index.html

`CVE ID [number](https://cve.mitre.org/about/faqs.html#what_is_cve_id) with four or more digits in the sequence number portion of the ID (e.g., "CVE-1999-0067", "CVE-2014-12345", "CVE-2016-7654321").`